apis/cache/v1beta1/cloudmemorystore_instance_types.go

```diff
@@ -187,6 +192,9 @@ type CloudMemorystoreInstanceObservation struct {
 	// for a given instance so should be checked before each import/export
 	// operation.
 	PersistenceIAMIdentity string `json:"persistenceIamIdentity,omitempty"`
+
+	// AuthString: AUTH string set on the instance.
+	AuthString string `json:"authString,omitempty"`
```

Comment:
If this is a sensitive value then we likely don't want it reflected in the status.

Reply:
oops! somehow I missed that. Should be fixed now.